docs/source/reference/routines.rst

```diff
@@ -26,3 +26,16 @@ These functions cover a subset of
    sorting
    statistics
    ext
+
+
+CUB backend for reduction routines
+----------------------------------
+Some CuPy reduction routines, including :func:`~cupy.sum`, :func:`~cupy.min`, :func:`~cupy.max`,
+:func:`~cupy.argmin`, :func:`~cupy.argmax`, and other functions built on top of them, can be
+accelerated by switching to the `CUB backend`_. The switch can be toggled on or off at runtime
```

Comment:
```suggestion
accelerated by switching to the `CUB`_ backend. The switch can be toggled on or off at runtime
```

Reply:
Haha, thanks. Just changed. I wasn't sure what's the best way to hyperlink this, as the name "CUB" is too short to make the link easily noticeable.